README.md

```diff
@@ -1,6 +1,15 @@
 [![Build status](https://ci.appveyor.com/api/projects/status/ryoqb5xj56jq0e04?svg=true)](https://ci.appveyor.com/project/GentenStudios/quartz-engine) [![Build Status](https://travis-ci.org/GentenStudios/quartz-engine.svg?branch=develop)](https://travis-ci.org/GentenStudios/quartz-engine)
 # QUARTZ ENGINE
 
+## Community
+Here is a link to our public discord where we generally collaborate and discuss the development of the engine.

```

Comment:
Change this to [Here is a link to our public discord server](blah link), where we collaborate and discuss the development of the engine.